temporal-sdk/src/main/java/io/temporal/internal/replay/ReplayWorkflowRunTaskHandler.java

```diff
@@ -163,7 +163,9 @@ public WorkflowTaskResult handleWorkflowTask(
       }
 
       handleWorkflowTaskImpl(workflowTask, historyIterator);
-      processLocalActivityRequests(wftHearbeatDeadline);
+      if (!context.isWorkflowMethodCompleted()) {
+        processLocalActivityRequests(wftHearbeatDeadline);
```

Comment:
Can you help me understand what is the exact behavior today? Do we start the LA but the server fails to accept the result marker command since workflows are done? Are there any concerns that people starting LAs at end of workflow but not waiting on them still expect that code to be reached and we are not calling it anymore?

Reply:
> Do we start the LA but the server fails to accept the result marker command since workflows are done

Correct

>Are there any concerns that people starting LAs at end of workflow but not waiting on them still expect that code to be reached and we are not calling it anymore?

I don't have any concern here because if someone was starting LAs at end of workflow but not waiting on them their workflow could never complete